apps/crn-server/serverless.ts

```diff
@@ -1189,6 +1193,21 @@ const serverlessConfig: AWS = {
         IS_CONTENTFUL_ENABLED: 'true',
       },
     },
+    sendSlackAlert: {
+      handler: './src/handlers/send-slack-alert.handler',
+      events: [
+        {
+          sns: {
+            arn: { Ref: 'TopicCloudwatchAlarm' },
+            topicName: '${self:custom.apiGateway5xxTopic}',
+          },
+        },
+      ],
+      environment: {
+        SLACK_WEBHOOK: SLACK_WEBHOOK!,
```

Comment:
what if you just pass an empty URL here instead, it will error out yeah? maybe thats what we should do? the aim is to have functions work the same way across all environments. So not only should we not conditionally disable parts of the infrastrcture, we should also not have anything like "if (env === "prod") { ..."

Reply:
i don't think we should use an empty url to configure if we send alerts to slack for the following reasons

- devs have remember that this is the way to disable it, compared to setting a boolean environment var
- only you have access to retrieve the hook url from slack 

Also i'm not sure i understand what's wrong with setting a flag based on the environment. But i can pass the value down from the workflow file.